master/pkg/model/task_config_policy_test.go

```diff
@@ -0,0 +1,42 @@
+package model
+
+import (
+	"testing"
+
+	"github.com/determined-ai/determined/proto/pkg/taskconfigpolicyv1"
```

Comment:
We can shorten to "configpolicy" or "configpolicies" if `taskconfigpolicy` feels too long. The only thing is, let's use "policy" or "policies" consistently. 

I think so far Saloni and I have been using "policies". Do you object to using the plural aka `policies`?

Reply:
I'm all good with plural! For the protobuf files though, it seems like all folders and `.proto` file names use singular and kind of like keeping consistent with that at least within the `proto` directory. WDYT?

Reply:
I like that; it makes sense to me.